frontend/packages/helm-plugin/src/catalog/utils/catalog-utils.tsx

```diff
@@ -113,9 +113,11 @@ export const normalizeHelmCharts = (
 
         const detailsDescriptions = [
           {
+            label: t('helm-plugin~Description'),
             value: <p>{description}</p>,
           },
           {
+            label: t('helm-plugin~README'),
```

Comment:
This change will result that the title `## README` is also shown when there is no README.

I don't find any chart without a README to confirm this, but I'm sure that was the reason to add it in the `HelmReadmeLoader`. You can see that README is now already displayed while it's loading.

Can you revert this change so that we are safe to not introduce another issue?